osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr55.json

```diff
@@ -65,5 +65,25 @@
         "access": "RW"
       }
     ]
-  }]
+  },
+    {
+      "tag": "ALARM_REGISTER_3",
+      "description": "Alarm register 3",
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.97.98.2.255",
```

Comment:
Ik zie deze niet in de 5.5 specs staan, zou best kunnen kloppen, maar wil graag weten waar deze staan in de specs

Reply:
This is specified in an addendum of spec SMR 5.5 and is added to the wiki.

Reply:
Check, good that it is added